src/jit/flowgraph.cpp

```diff
@@ -20637,6 +20646,12 @@ unsigned BBPredsChecker::CheckBBPreds(BasicBlock* block, unsigned curTraversalSt
     }
 
     unsigned blockRefs = 0;
+
+    CompAllocator memAllocator(comp, CMK_DebugOnly);
+
+    typedef JitHashTable<unsigned, JitSmallPrimitiveKeyFuncs<unsigned>, bool> VisitedPreds;
```

Comment:
Hmm, maybe I should make a proper `JitHashSet`... There are other places that use this workaround, including non debug code.